spec/Section 3 -- Type System.md

```diff
@@ -388,9 +388,10 @@ Scalar types represent primitive leaf values in a GraphQL type system. GraphQL
 responses take the form of a hierarchical tree; the leaves of this tree are
 typically GraphQL Scalar types (but may also be Enum types or {null} values).
 
-GraphQL provides a number of built-in scalars which are fully defined in the
-sections below, however type systems may also add additional custom scalars to
-introduce additional semantic meaning.
+GraphQL provides a number of
+[built-in scalars](#sec-Appendix-Built-in-Definitions.Scalars) which are fully
```

Comment:
I don't think this should reference an appendix. Following the definitions below is fine. Note that Appendix B isn't actually referenced anywhere in the spec text.

Reply:
Same for the below references, I think they also should be removed. The appendix doesn't _define_ the built in directives, it merely catalogues them in one place.